pages/checkout.js

```diff
@@ -79,20 +90,19 @@ module.exports = {
 
   deliveryPriceText: { xpath: "//strong[text()='Flat Shipping Rate:']/ancestor::tr/td[2]" },
   totalPriceText: { xpath: "//strong[text()='Total:']/ancestor::tr/td[2]" },
+  unavailablItem: { xpath: `//*[@id="checkout-cart"]/div[contains(.,'Products marked with *** are not available in the desired quantity or not in stock!')]` },
```

Comment:
add word at the end of name to identify what kind of item is found by locator, e.g. Text, Button, Field, Option